lib/cutest.rb

```diff
@@ -148,19 +148,19 @@ def test(name = nil, &block)
     cutest[:test] = nil
   end
 
-  # Assert that value is not nil or false.
-  def assert(value)
-    flunk("expression returned #{value.inspect}") unless value
+  # Assert that test is a true value.
+  def assert(test)
+    flunk("expression returned #{test.inspect}") unless test
     success
   end
 
-  # Assert that two values are equal.
-  def assert_equal(value, other)
-    flunk("#{value.inspect} != #{other.inspect}") unless value == other
+  # Assert that expected and actual values are equal.
+  def assert_equal(expected, actual)
```

Comment:
@djanowski So, `value` and `other` provide no clue to which are the expected and actual values. From your comment, I assume you prefer `actual, expected`. This actually was noted while doing a code reading exercise at the Ruby Montevideo meetup. This is Ruby's standard library method signature for `assert_equal`:

http://www.rubydoc.info/gems/test-unit/2.2.0/Test/Unit/Assertions#assert_equal-instance_method

Do you prefer to swap the args and re-push or, discard?